schema-derivatives-perpfutures.graphql

```diff
@@ -1803,3 +1902,39 @@ type PositionSnapshot @entity(immutable: true) {
   " Timestamp of this snapshot "
   timestamp: BigInt!
 }
+
+type _ActivityHelper @entity {
```

Comment:
I don't think it is necessary to have different fields for daily/hourly. Couldn't we just have `activeUsers` for example and then that will either be for daily or hourly depending on the id being used?

Reply:
yeah, that makes sense. I will implement that.